github/provider.go

```diff
@@ -308,13 +318,21 @@ func providerConfigure(p *schema.Provider) schema.ConfigureFunc {
 		}
 		log.Printf("[DEBUG] Setting read_delay_ms to %d", readDelay)
 
+		parallelRequests := d.Get("parallel_requests").(bool)
+		isGithubDotCom, _ := regexp.MatchString("^https://api.github.com", baseURL)
```

Comment:
What's the reason we're ignoring this error here?

Reply:
From the doc was triggered  only when setting a malformed regexp so I skipped it.I am going to add the error check.

Reply:
Thank you!